### PR TITLE
Hey there Scala!!

### PR DESCRIPTION
**Pull request by tkadlec**

### CODE REVIEW

File: _docs_using-snyk/04_test.md [hunk 1]
```diff
@@ -10,7 +10,7 @@ title: Test
 <p><code>snyk test</code> takes stock of all the local dependencies and queries the snyk service for related known vulnerabilities. It displays the found issues along with additional information. For Node.js projects, it also suggests remediation steps.</p>
 
 <p>When <code>snyk test</code> runs, it tries to detect the appropriate file for your project by looking for the following files, in this order:</p>
-<ol><li>yarn.lock</li><li>package.json</li><li>Gemfile</li><li>Gemfile.lock</li><li>pom.xml</li><li>requirements.txt</li></ol>
+<ol><li>yarn.lock</li><li>package.json</li><li>Gemfile</li><li>Gemfile.lock</li><li>pom.xml</li><li>requirements.txt</li><li>build.sbt</li></ol>
```
Comment (adrukh) [inline _docs_using-snyk/04_test.md]: If we're being really detailed, `build.gradle` should be on this list right before `build.sbt`

Comment (tkadlec) [inline _docs_using-snyk/04_test.md]: Ok. Will add.

File: _layouts/docs.html [hunk 2]
```diff
@@ -80,6 +81,16 @@ <h1 id="documentation">Documentation</h1>
           </ul>
           {% endif %}
         </li>
+        <li class="nav-tree__item nav--docs__item nav--docs__python">
+          <a href="/docs/scala/" class="nav-tree__item__link">Snyk for Scala</a>
+          {% if page.section == "python" %}
```
Comment (darscan) [inline _layouts/docs.html]: Is this correct? Conditional refers to python, but block is for Scala, no?

Comment (tkadlec) [inline _layouts/docs.html]: Whoopsies! Fixed.

**Pull request merged**